config.test.yaml

```diff
@@ -8,7 +8,7 @@ default_project: &default_project
     - path: projects/wmf_default.yaml
       options: &default_options
         table:
-          backend: cassandra
+          backend: sqlite
```

Comment:
This is clearly from some rebasing weirdness or github being buggy..

Could you try to rebase again and force-push into your branch? 

like on master `git pull upstream master; git checkout pcs/media; git rebase master; git push -f origin pcs/media`

Reply:
Ok, rebased. It's weird that I have to do a `-f`.